app/assets/javascripts/browse_everything/behavior.js.coffee

```diff
@@ -100,9 +104,11 @@ $ ->
           else
             if action == "1"
               $(this).addClass('ev-selected')
+              $(this).find('.ev-select-file').prop('checked', true)
```

Comment:
If this code was in the selectFile method I think it would only need to be in one place, not in both.